src/coresyftools/coresyftools/tests/test_tool.py

```diff
@@ -312,3 +312,39 @@ def test_can_use_custom_manifest_name(self):
         self.assertEqual(tool.manifest['name'], 'MyFairTool')
         os.remove(manifest_file_name)
 
+    def test_zip_output(self):
+        """Test the method to compress an output folder"""
+        tool = CoReSyFTool(self.runfile, 'manifest.json')
+        tool.write_to_zipfile('coresyftools/tests/tool6/myoutput/')
```

Comment:
Where is such directory?

Reply:
I forgot to commit the test tool. However, due to a change I made, the test that uses this tool now automatically creates the directory.